tools/python/find_optimizer_opset_version_updates_required.py

```diff
@@ -71,28 +71,29 @@ def get_multiline_call_args_from_file(filename, function_or_declaration):
 
     results = []
     with open(filename) as f:
-        cur_line = None
+        current_args = None
 
         for line in f:
-            if not cur_line:
+            if not current_args:
                 # look for new match
                 start = line.find(function_or_declaration)
                 if start != -1:
-                    cur_line = line
+                    # include the '(' in case that is the last character in the line
+                    current_args = line[start:].strip()
```

Comment:
Looks like find returns the start position of the substring. line[start:] will include `function_or_declaration` as well. is this intended?

```
>>> "abcd".find("bc")
1
```

Reply:
Good catch. The script works but only as the first argument wasn't used in the calling code. Reworked it to be a bit more explicit.